db/migrate/20200819173237_remove_printer_id.rb

```diff
@@ -0,0 +1,14 @@
+class RemovePrinterId < ActiveRecord::Migration
+  def up
+    User.find_each do |user|
+      if user.printer_id
+        PrintersUser.create(user: user, printer_id: user.printer_id, tag: "stock")
```

Comment:
Would it be better for this to be a rake task? Imagine in the future if we don't have a `PrintersUser` model anymore... this would fail